src/js/providers/html5.js

```diff
@@ -703,6 +659,10 @@ define([
             }
         };
 
+        this.playbackComplete = function() {
+            clearTimeout(_playbackTimeout);
+        };
```

Comment:
Instead of adding `provider.playbackComplete`, just clear this on ended. We don't need it in the providers.